tests/system/providers/amazon/aws/example_redshift.py

```diff
@@ -236,10 +203,8 @@ def create_connection(conn_id_name: str, cluster_id: str):
         wait_cluster_paused,
         resume_cluster,
         wait_cluster_available_after_resume,
-        set_up_connection,
         create_table_redshift_data,
         insert_data,
-        drop_table,
```

Comment:
Wasn't this meant to be replaced with another implementation that doesn't use the SQL connection?

Reply:
I realized this task is useless since we are removing the cluster after